src/main/java/graphql/schema/impl/GraphQLTypeCollectingVisitor.java

```diff
@@ -1,12 +1,26 @@
-package graphql.schema;
```

Comment:
Again I moved this to a schem.impl package just to clean things up. Its kinda of a private thing and not general purpose